ConsoleAppAzureServices/ConsoleAppAzureServices/Program.cs

```diff
@@ -1,47 +1,49 @@
-﻿using System;
+﻿using Azure;
+using Azure.Identity;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.Resources.Models;
+using Azure.Security.KeyVault.Secrets;
+using System;
 using System.Threading.Tasks;
-using Microsoft.Azure.KeyVault;
-using Microsoft.Azure.Management.ResourceManager;
-using Microsoft.Azure.Services.AppAuthentication;
-using Microsoft.Rest;
 
 namespace ConsoleAppAzureServices
 {
     class Program
     {
         static void Main()
         {
-            AzureServiceTokenProvider azureServiceTokenProvider = new AzureServiceTokenProvider();
+            InteractiveBrowserCredential credential = new InteractiveBrowserCredential();
```

Comment:
We recommend people use `DefaultAzureCredential`, which may prompt with the browser, but will automatically use Azure CLI, Az PowerShell modules, VS, or VSCode credentials if already logged in.

Reply:
Agreed, `DefaultAzureCredential` should be used for samples unless those samples are specifically demonstrating other forms of authentication.